app/src/main/java/com/example/habitsmasher/ui/profile/ProfileFragment.java

```diff
@@ -45,16 +45,16 @@
     private static final String USER_ID_SHARED_PREF_TAG = "userId";
     private static ArrayList<String> EMPTY_FOLLOWER_LIST = new ArrayList<>();
     private static ArrayList<String> EMPTY_FOLLOWING_LIST = new ArrayList<>();
+    private static ArrayList<String> EMPTY_REQUEST_LIST = new ArrayList<>();
 
     private ProfileFragment _fragment = this;
     private ImageView _userImageView;
     private Bitmap _userImage;
+    private User user = new User();
```

Comment:
Why do we need this?

Reply:
When deleting the sample User, there was a method that actually called the variable which the sample user was stored in:

`user.setUsername(sharedPref.getString(USERNAME_SHARED_PREF_TAG, "user"));`

I'm not too familiar with the ProfileFragment class so I added that so we don't get NPEs